### PR TITLE
Create Game of Clue - Terrorist Attack.md

### PR DESCRIPTION
**Pull request by sarikasu**

### CODE REVIEW

File: tasks/Game of Clue - Terrorist Attack.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+2. Subjects could take part in up to three runs of the experiment. Within any given run, all subjects had the same level of experience with the experiment. That is, if a subject were taking part in a second run, then all of the other subjects in that run would also be taking part for the second time. Holding this experience constant eliminated the possibility of spurious correlations between learning and network structure. In the multivariate analyses, we include a round variable that indicates the experience of the participants in number of runs through the experiment.
+> Before each run began, a network treatment was chosen at random and study subjects were randomly assigned to a position in that network that was uncorrelated to their physical location in the laboratory. Participants were assigned a pseudonym to further obscure their identities from each other; the pseudonyms were shuffled before each round.
+4. Participants were given two clues at the start of each round and were allowed to search for more clues once per minute.
```
Comment (katelynbottcher) [inline tasks/Game of Clue - Terrorist Attack.md:19]: They could share clues they had one at a time with one or more of their neighbors, and could add free-text annotations to these shared clues

File: tasks/Game of Clue - Terrorist Attack.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+> Before each run began, a network treatment was chosen at random and study subjects were randomly assigned to a position in that network that was uncorrelated to their physical location in the laboratory. Participants were assigned a pseudonym to further obscure their identities from each other; the pseudonyms were shuffled before each round.
+4. Participants were given two clues at the start of each round and were allowed to search for more clues once per minute. 
+5. The initial distribution of clues was not correlated with outcomes. Each clue was only relevant to one subproblem. Some clues contained useless or misleading information. 
+6. Subjects had to combine multiple clues to conclusively arrive at the correct answer. The number of clues necessary to solve a single subproblem ranged from 2 to 10, with a median of 5 and a mean of 5.3.
```
Comment (katelynbottcher) [inline tasks/Game of Clue - Terrorist Attack.md:21]: Each participant typed their solution into the separate spaces given for each subproblem, and could check their neighbors solutions at any time and any number of times.

File: tasks/Game of Clue - Terrorist Attack.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+# References
+Main Paper: https://pubsonline.informs.org/doi/full/10.1287/orsc.2015.0980#_i34
+
+Supplemental Materials: https://pubsonline.informs.org/doi/suppl/10.1287/orsc.2015.0980/suppl_file/orsc.2015.0980-sm-online_appendix_.pdf
+
+# Stimuli
+## The visual components
+See supplemetal materials PDF for images of computer interface 
+
+
```
Comment (katelynbottcher) [inline tasks/Game of Clue - Terrorist Attack.md:13]: For materials for other alternate versions:
- There were four 16-person network treatments
- 1) Caveman (CAVE): contained four four-person cliques
- 2) Hierarchy (HIER): four four-person cliques arranged in a conventional centralized structure
- 3) Rewired Caveman (RCAVE): removed links from the caveman network, then added links that created shortcuts through the network
- 4) Ring (RING): a network that was neither clustered nor centralized

Comment (sarikasu) [reply tasks/Game of Clue - Terrorist Attack.md:13]: I thought these were more of the treatment groups within the experiment, then they compared results across the different network structures. I can add in to the procedure section but I'm not sure they would be considered separate experiments?

Comment (katelynbottcher) [reply tasks/Game of Clue - Terrorist Attack.md:13]: sounds good!

**Review by katelynbottcher — APPROVED** [review]

**Pull request merged**